Content.Server/_RMC14/Xenonids/Burrow/XenoBurrowSystem.cs

```diff
@@ -290,6 +290,8 @@ private void SetBurrow(Entity<XenoBurrowComponent> xenoBurrower, bool active)
                 Dirty(action.Id, actComp);
             }
 
```

Comment:
This needs to call physics system try set body type kinematic controller after unanchor, or the xeno will be launched to the moon by explosions as happened with the queen and defenders